src/import.ts

```diff
@@ -1,5 +1,6 @@
 import JSZip from "jszip";
 import { Contact } from "./contact";
+// import { parse } from "@fortawesome/fontawesome-svg-core";
```

Comment:
?

Reply:
NextJS wouldn't build with this uncommented. Idk either